README.rst

```diff
@@ -72,7 +72,7 @@ Here's a full example with the libvirt provider:
    driver:
      name: vagrant
      provider:
-       # Can be any supported provider (VBox, Parallels, libvirt, etc)
+       # Can be any supported provider (VirtualBox, Parallels, libvirt, etc)
```

Comment:
If you're at changing provider name because VBox is not correct, VirtualBox and Parallels are not correct. Technically, they all should be lower case.

Reply:
You fixed that with https://github.com/ansible-community/molecule-vagrant/pull/94 @apatard :)

Reply:
no, I've made it work with upper cases to avoid such small typos but it doesn't make it the right value :)

Reply:
good point